docs/DEPLOYMENT.md

```diff
@@ -146,25 +175,106 @@ data:
   GH_CLIENT_ID: GH OAuth Client ID   <<< update this
 ```
 
-#### Apply the manifests
+### Apply the manifests
 
-```
+```bash
 kubectl apply -f config
 ```
 
-#### Ensure pods are up and running
+### Ensure pods are up and running
 
-```
+```bash
 kubectl get pods
 ```
 
+### Setup Route/Ingress
+
+After the deployment is done successfully, we need to expose the URL to access the UI.
+
+- If deploying on OpenShift:-
+
+  ```bash
+  oc apply -f config/openshift/
+  ```
+
+- If deploying on Kubernetes:-
+
+  - Create the secret containing tls cert and tls key
+
+    ```bash
+    kubectl create secret tls ui-hub-tekton-dev-tls --cert=path/to/cert/file --key=path/to/key/file
+    ```
+
+  - Apply the Ingress
+
+    ```bash
+    kubectl apply -f config/post-deploy
+    ```
+
+### Verify if UI route is accessible
+
 If on openshift verify if the ui route is accessible
 
-Open: oc get routes ui --template='https://{{ .spec.host }} '
+Open: `oc get routes ui --template='https://{{ .spec.host }} '`
+
+## Deploying for the first time?
+
+> Note: Make sure you have added your name in the scopes present [config.yaml][config.yaml]
+
+1. Open currently deployed hub in the browser
+2. Login into the Hub
+3. After successful login, now open the terminal and follow below steps.
+4. Exec into the database pod
+   ```bash
+   kubectl exec -it <db-pod-name> bash
+   ```
+5. Open PostgreSQL terminal by executing the command
+
+   ```bash
+   psql -U postgres
+   ```
+
+   You'll see the terminal changed to
+
+   ```bash
+   $ psql -U postgres
+   psql (13beta3 (Debian 13~beta3-1.pgdg100+1))
+   Type "help" for help.
+
+   postgres=#
+   ```
+
+6. Connect to Hub database by executing
+   ```sql
+   postgres=# \c hub
+   ```
+7. Execute the SQL query
+
+   ```sql
+   postgres=# delete from configs;
+   ```
+
+8. Quit the database pod by first executing
+
+   ```sql
+   postgres=# \q
+   ```
+
+   and then `exit`.
+
+9. Delete the API pod by executing the following command
+   ```bash
+   oc delete pod <pod-name>
+   ```
+   This will delete the previously created pod and spin a new pod which will add the scopes in the DB
+10. Repeat the steps 1-2
+11. Copy the Hub Token from the UI and add that in [catalog refresh step](###In-case-of-refreshing-the-catalog)
```

Comment:
I think it is not getting the reference correctly

Reply:
resolved :+1: